fastlane/Fastfile

```diff
@@ -222,23 +220,48 @@ platform :ios do
     )
 
     UI.important('Pushing changes to remote and configuring the release on GitHub')
-    unless options[:skip_confirm] || UI.confirm('Do you want to continue?')
-      UI.user_error!("Terminating as requested. Don't forget to run the remainder of this automation manually.")
-    end
+    UI.user_error!("Terminating as requested. Don't forget to run the remainder of this automation manually.") unless skip_confirm || UI.confirm('Do you want to continue?')
 
     push_to_git_remote(tags: false)
 
+    # Protect release/* branch
     copy_branch_protection(
       repository: GITHUB_REPO,
       from_branch: DEFAULT_BRANCH,
       to_branch: "release/#{new_version}"
     )
-    set_milestone_frozen_marker(
-      repository: GITHUB_REPO,
-      milestone: new_version
-    )
 
-    check_pods_references
+    begin
+      # Add ❄️ marker to milestone title to indicate we entered code-freeze
+      set_milestone_frozen_marker(
+        repository: GITHUB_REPO,
+        milestone: new_version
+      )
+
+      # Move PRs to next milestone
+      moved_prs = update_assigned_milestone(
+        repository: GITHUB_REPO,
+        from_milestone: new_version,
+        to_milestone: release_version_next,
+        comment: "Version `#{new_version}` has now entered code-freeze, so the milestone of this PR has been updated to `#{release_version_next}`."
+      )
```

Comment:
These 2 calls need to be swapped, otherwise due to a latency in GitHub updating its database after a milestone rename from `set_milestone_frozen_marker`, the `update_assigned_milestone` will fail to find any PR as results of the search query for a couple of seconds. See also https://github.com/woocommerce/woocommerce-android/pull/12265#discussion_r1722105708

Reply:
Thanks for the heads up 👍  I've changed it in this PR as well as in the other ones.